recipe/meta.yaml

```diff
@@ -20,16 +19,15 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch     # [(not win) and (py==311)]
-    - m2-patch  # [win and (py==311)]
+    - patch     # [(not win)]
+    - m2-patch  # [win]
```

Comment:
FYI, in the future, the `upd` C library will be the main backend instead of C++ https://github.com/protocolbuffers/protobuf/tree/v23.4/python#implementation-backends but conda-forge still uses C/C++